pyproject.toml

```diff
@@ -95,21 +94,41 @@ build-backend = "poetry_dynamic_versioning.backend"
 [tool.poetry.extras]
 docs = ["mkdocs-material"]
 
+
+# scripts should have click clis
+# click parameters/options should be written out with hyphens
+# click parameters/options should use consistent names
+# scripts should be listed here in tool.poetry.scripts
+# use underscores to delimit file name
+# use hyphens to delimit CLI alias
+# scripts/aliases should be illustrated in a Makefile
+# Makefile targets should make use of $< etc for input and $@ for output
+# input and output files should use hyphens as delimters in their names
+# check usages of these features in other NMDC code before changing
+# be intentional about output: most should go in assets unless its really large or likely to change independently of the schema, like MOngoDB summaries
+# cli entry points should probably all be main or cli unless there's a compelling reason to do otherwise
+
+
 [tool.poetry.scripts]
+analyze-mentions-of-ids = "src.scripts.analyze_mentions_of_ids:main"
 anyuri-strings-to-iris = 'nmdc_schema.anyuri_strings_to_iris:expand_curies'
-build-datafile-from-api-requests = "nmdc_schema.build_datafile_from_api_requests:main"
-class-sparql = "nmdc_schema.class_sparql:main"
-fetch-nmdc-schema = "nmdc_schema.nmdc_data:get_nmdc_jsonschema"
-generate-import-slots-regardless = "nmdc_schema.generate_import_slots_regardless:main"
-get-mixs-slots-matching-slot-list = "nmdc_schema.get_mixs_slots_matching_slot_list:main"
-get-mixs-slots-used-in-schema = "nmdc_schema.get_mixs_slots_used_in_schema:main" # see nmdc_schema/generate_import_slots_regardless.py
-get-slots-from-class = "nmdc_schema.get_slots_from_class:main" # see tests/test_getters.py
-get-study-related-records = "src.scripts.nmdc_database_tools:cli"
-migration-recursion = 'nmdc_schema.migration_recursion:main'
+check-examples-class-coverage = "src.scripts.check_examples_class_coverage:main"
+class-instantiation-counts = "src.scripts.class_instantiation_counts:process_data"
 create-migrator = 'nmdc_schema.migrators.cli.create_migrator:create_migrator'
-nmdc-data = "nmdc_schema.nmdc_data:cli"
-nmdc-version = "nmdc_schema.nmdc_version:cli"
-slot-range-type-reporter = "src.scripts.slot_range_type_reporter:cli"
-get-study-id-from-filename = "src.scripts.get_study_id_from_filename:main"
+database-to-json-list-files = "src.scripts.database_to_json_list_files:split_yaml_to_json"
+date-created-blank-node = "src.scripts.date_created_blank_node:print_timestamp" # prints a RDF Tutrle string, for copying into other RDF/TTL files. may need some paramterizastion.
```

Comment:
There is a typo in the word "Turtle". I don't consider this to be a deal-breaker given the task at hand.